docs/index.md

```diff
@@ -4,6 +4,145 @@ layout: default
 
 # Zarr benchmarks
 
+## Introduction
+
+This page contains benchmarks for the best options to use when creating Zarr
+arrays.
+
+[Zarr](https://zarr.dev/) is a specification for storing array data. When
+creating a Zarr dataset there are several confugration options you can set,
+which all have an impact on data compression, read times, and write times. This
+project provides guidance on how array creation options affect data size and
+read/write performance.
+
+### Zarr versions
+
+At the time of writing tools for working with version 3 of the Zarr array format
+were still works in progress, so all testing was done reading/writing version 2
+Zarr arrays. We expect performance to be similar when reading/writing version 3
+Zarr arrays that are not sharded, because they should write identical data to
+their corresponding version 2 Zarr array.
+
+This is part of the [HEFTIE project](https://github.com/HEFTIEProject).
+
+## Executive summary
+
+- **Software**: _[tensorstore](https://google.github.io/tensorstore/)_ is faster
+  than [_zarr-python_ version 3](https://zarr.readthedocs.io/en/stable/) is
+  faster than [_zarr-python_ version 2](https://zarr.readthedocs.io/en/v2.18.5/)
+  (for both reading and writing data).
+- **Compressor**: _blosc-zstd_ provides the best compression ratio.
+- **Compression level**: Setting compression levels beyond ~3 results in
+  slightly better data compression but much longer write times. Compression
+  level does not affect read time.
+- **Other compressor options**: Setting the _shuffle_ option increases data
+  compression with no adverse effect on read/write times.
+
+## Configuration
+
+Data used for benchmarking is available at
+[https://doi.org/10.5281/zenodo.15544055]().
+
+### Dense image data
+
+- **Image data**: A **(shape)** shaped image of a human heart with _uint16_ data
+  type.
+- **Dense label data**: A **(shape)** shaped **description** with **dtype** data
+  type.
+- **Sparse label data**: A **(shape)** shaped **description** with **dtype**
+  data type.
+
+### Default configuration
+
+Unless stated as being varied, the default confugration used was:
+
+- **Chunk size** = 128 x 128 x 128
+- **Compressor** = blsoc-zstd
+- **Shuffle** = "shuffle"
+- **Compression level** = ???
```

Comment:
@ruaridhg or @K-Meech could you update this with the correct default configuration.

Reply:
I can update this 👍